src/framework/db/core.clj

```diff
@@ -77,9 +76,8 @@
   When no parameter given, then resolves database specs from configuration"
   [config]
   (let [db-spec (:framework.db.storage/postgresql config config)
-        init-sql (when-let [sql (some-> (:init-script db-spec) slurp)] [sql])
         db-instance (case (:deployment db-spec)
```

Comment:
maybe it should be removed here, and advice to use docker-postgres! separately, on app startup.